src/services/balancer/entities/pools.ts

```diff
@@ -0,0 +1,157 @@
+import { PoolType } from '@/services/balancer/subgraph/types';
+
+import {
+  Vault__factory,
+  WeightedPoolFactory__factory,
+  WeightedPool__factory
+} from '@balancer-labs/typechain';
+import { Contract } from '@ethersproject/contracts';
+import { TransactionResponse, TransactionReceipt, Web3Provider } from '@ethersproject/providers';
+import { configService } from '@/services/config/config.service';
+import BigNumber from 'bignumber.js';
+import { BigNumber as EPBigNumber } from '@ethersproject/bignumber';
+import { sendTransaction } from '@/lib/utils/balancer/web3';
+import { defaultAbiCoder } from '@ethersproject/abi';
+import { TokenWeight } from '@/composables/pools/usePoolCreation';
+import { toNormalizedWeights } from '@balancer-labs/balancer-js';
+import { scale } from '@/lib/utils';
+
+type Address = string;
```

Comment:
I think this is an unnecessary type.

Reply:
So I was thinking I'd like this to be more standard, rather than just in this location. As there's many places where a string could mean an address, a stringified big number, or something else, and I thought it would be useful to type that explicitly so there's no confusion.